config/routes.rb

```diff
@@ -2,9 +2,11 @@
 
 Rails.application.routes.draw do
   # Pages
+  get '', to: 'pages#thankyou'
   get '/live', to: 'pages#live'
   get '/comingsoon', to: 'pages#comingsoon'
-  get '/home', to: 'pages#index'
+  get '/home', to: 'pages#thankyou'
```

Comment:
I recommend keeping this as index -- makes it easier to work on next year's homepage without making it the actual homepage. Otherwise you can just delete this line

```suggestion
  get '/home', to: 'pages#index'
```

Reply:
Chris wants this as the main brickhack.io page...unless I misunderstood what you were trying to say here

Reply:
Right, however the `get '/home'` line only defines visiting https://brickhack.io/home, not the actual homepage. By keeping this line as `get '/home', to: 'pages#home'`:
* https://brickhack.io (and http://localhost:3000/) will remain the correct "thank you" homepage
* https://brickhack.io/home (and http://localhost:3000/home) will remain the "regular" homepage. 

Makes it easy to work on the "regular" homepage by developing on/previewing the `/home` URL while keeping the actual `/` URL pointed to the thank you page